ntoskrnl/mm/ARM3/miarm.h

```diff
@@ -786,7 +786,8 @@ MI_MAKE_HARDWARE_PTE_KERNEL(IN PMMPTE NewPte,
 
     /* Check that we are not setting valid a page that should not be */
     ASSERT(ProtectionMask & MM_PROTECT_ACCESS);
-    ASSERT((ProtectionMask & MM_GUARDPAGE) == 0);
+    ASSERT(((ProtectionMask <= MM_WRITECOMBINE || ProtectionMask >= MM_OUTSWAPPED_KSTACK) &&
```

Comment:
These have to be (less/greater than) _**OR equal**_ ?
NB: The `ProtectionMask <= MM_WRITECOMBINE` equivalent to `ProtectionMask <= MM_NOACCESS`.